neqo-crypto/build.rs

```diff
@@ -285,13 +284,69 @@ fn main() {
         };
         dynamic_link(libs);
     }
+    includes
+}
+
+fn setup_for_gecko() -> Vec<PathBuf> {
+    let mut includes: Vec<PathBuf> = Vec::new();
+    let libs = if env::consts::OS == "windows" {
+        &["nssutil3.dll", "nss3.dll", "ssl3.dll"]
+    } else {
+        &["nssutil3", "nss3", "ssl3"]
+    };
+    dynamic_link(libs);
+
+    match env::var_os("MOZ_TOPOBJDIR") {
+        Some(path) => {
+            let nsprinclude = PathBuf::from(path.clone())
+                .join("dist")
+                .join("include")
+                .join("nspr");
+            includes.push(nsprinclude);
+            let nssinclude = PathBuf::from(path.clone())
+                .join("dist")
+                .join("include")
+                .join("nss");
+            includes.push(nssinclude);
+            for i in &includes {
+                println!("cargo:include={}", i.to_str().unwrap());
+            }
+        }
+        None => {
+            println!("cargo:warning={}", "MOZ_TOPOBJDIR should be set by default, otherwise the build is not guaranted to finish.");
```

Comment:
```suggestion
            println!("cargo:warning={}", "MOZ_TOPOBJDIR should be set by default, otherwise the build is not guaranteed to finish.");
```